Automattic/wp-calypso
# Logos / Colors: add Nextdoor logo and colors

Related to #78562

> **Note**
> This is currently against the branch for the PR above. We could either merge it into that branch, or wait and merge it against `trunk` later.

## Proposed Changes

Start using Nextdoor in the Calypso interface after updating the Social Logos package to be able to benefit from the new  logo.

## Testing Instructions

* Open the Calypso live link below, and go to `**.calypso.live/devdocs/design/social-logo`: you should see the new logos.
* To see the change in the sharing buttons settings, this will need to be merged first: https://github.com/Automattic/jetpack/pull/31537
* Until then, you should be able to test by looking at a site where you have checked out the branch from that PR.
    * Go to `https://wordpress.com/marketing/sharing-buttons/yoursite.com`
    * Try to add a Nextdoor sharing button.

## Pre-merge Checklist
